ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Nagios_Plugins/tasks/nagios_RedHat.yml

```diff
@@ -41,6 +42,7 @@
   when:
     - ansible_distribution_major_version != "6"
     - not (ansible_distribution_major_version == "7" and ansible_architecture == "s390x")
+    - not (ansible_distribution_major_version == "10")
```

Comment:
Same comment about `>=`
Also unrealted to this PR but does the s390x conditional need to be specific to 7?

Reply:
Yes, for other architectures the nagios plugins install don't have that link, but for some reason on s390x the nagios packages create it. Fixed the >=